code/datums/uplink_item.dm

```diff
@@ -954,7 +936,7 @@ var/list/uplink_items = list()
 	will allow you all the mobility of a loose fitting uniform without sacrificing armoring. Additionally the suit is collapsible, small enough to fit within a backpack. \
 	Nanotrasen crewmembers are trained to report red space suit sightings; these suits in particular are known to drive employees into a panic."
 	item = /obj/item/clothing/suit/space/hardsuit/syndi
-	cost = 7
+	cost = 6
```

Comment:
I guess if we see more traitors going full ham with weapons I guess this becomes more viable? Why not.